scripts/generator.js

```diff
@@ -42,7 +43,8 @@ async function renderComponent(options) {
 
     // Provide the template through this function instead of the URL,
     // as it will contain large SVG files.
-    await page.exposeFunction("getTemplate", templateId => getTemplate({ id: templateId }, "svg"));
+    await page.exposeFunction("getTemplate", templateId => getTemplate({ id: templateId }));
+    await page.exposeFunction("serverLog", log);
```

Comment:
Removed unnecessary second argument to getTemplate. Also exposed a function to the poster app that outputs debug messages in the server console, which was of huge help when debugging.